src/jdk.crypto.cryptoki/share/native/libj2pkcs11/pkcs11t.h

```diff
@@ -102,9 +102,14 @@ typedef CK_ULONG CK_NOTIFICATION;
 #define CKN_SURRENDER           0UL
 #define CKN_OTP_CHANGED         1UL
 
+typedef CK_ULONG CK_PROFILE_ID;
```

Comment:
There is already a typedef for CK_PROFILE_ID further down in this file.

Reply:
oh, missed it. Thanks